bigip/resource_bigip_ltm_pool.go

```diff
@@ -18,13 +18,15 @@ func resourceBigipLtmPool() *schema.Resource {
 		Update: resourceBigipLtmPoolUpdate,
 		Delete: resourceBigipLtmPoolDelete,
 		Exists: resourceBigipLtmPoolExists,
+		Importer: &schema.ResourceImporter{
+			State: resourceBigIpLtmPoolImporter,
+		},
 
 		Schema: map[string]*schema.Schema{
 			"name": &schema.Schema{
 				Type:         schema.TypeString,
 				Required:     true,
 				Description:  "Name of the pool",
-				ForceNew:     true,
```

Comment:
Is changing the name of an existing resource always allowed in via the API? If so we should remove these across the board for consistency. If not or unsure, can you please add a test case to validate changing the name of an existing pool is supported.